src/container/App.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './App.css';
+import Display from '../components/Display/Display';
+import ButtonPanel from '../components/ButtonPanel/ButtonPanel';
+import calculate from '../logic/calculate';
+
+class App extends React.Component {
```

Comment:
There's a nicer syntax provided by a babel plugin (something w/ the name `transform-class-property`). This plugin is included in `create-react-app` by default. Thus you can take advantage of it. This nicer syntax will save you from having to use the `constructor` (and binding your methods):

```javascript

class App extends React.Component {
  state = {
    calculation: '0',
      total: '',
      current: '',
      operator: ''
  }

  handleClick = (btnName) => this.setState(prevState => calculate(prevState, btnName);
 // see my annotation below on handleClick

  render () {
     // same stuff
   }
}
```

Reply:
I am trying to implement this, but the way it sets state in this function doesn't work after refactoring giving an unexpected token error

Reply:
Ok that's weird because I tried it and it wasn't giving me any errors. Anyway, you aren't required to change this. That was just a recommendation! I encourage you though to still try it and get used to that newer syntax :)